### PR TITLE
Integration of extension with Wayback Machine's site search with auto-suggest and display of information inside extension popup

### PR DESCRIPTION
**Pull request by abhidas17695**
The text field allows searching for both a keyword and a URL. 
If it is a URL then all archives of that URL are displayed in the form of a calendar as in the website.
capture1

If it is a keyword then all webpages matching the keyword are listed

  
capture2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      <div id="disp_search"></div>
+      
+      <div id="disp_months" class="hidden"></div>
+      <div id="disp_days" class="hidden"></div>
+      <div id="disp_time" class="hidden"></div>
+  </body>
+</html>
+<script src="scripts/jquery.js"></script>
+<script src="scripts/popup.js"></script>
+<script src="scripts/bootstrap.min.js"></script>
```
Comment (rgpt) [inline index.html]: Jquery is not needed in this project due to performance issues

Comment (abhidas17695) [inline index.html]: I agree on that

**Comment by rgpt** [comment]
I think you should send pull request to respective version and not master

**Comment by abhidas17695** [comment]
Thanks for the review @rgpt . I changed the base branch of the PR as you suggested.

**Comment by abhidas17695** [comment]
The latest commit resolved the merge conflicts caused due to recent merges into the base repo. All functionalities of the previous commit are maintained.
newsearchmain

To search for a URL on Twitter or to share a URL on social media , the input text bar used for both purposes is the same. To search on Twitter just enter the URL , the date range and click on the "Search" button. To share on social media just enter the URL and click any of the social media buttons.

**Comment by abhidas17695** [comment]
Latest commit resolved merge conflicts. I have changed the UI a bit.  I have enclosed the text bar for the Twitter search and social media sharing buttons inside the panel. Since they all work together it makes more sense to the user if they are clubbed together instead of being separated  by the ' Save now ' , ' First version ' and ' Recent version ' buttons. I have provided a separate text bar for Site Search. Auto-suggestions will be visible as the user types just like in the Site Search bar in the website. The search results will also be visible and accessible inside the extension itself to provide maximum utility to the user. The user will thus not have to shuffle between tabs. 
new2 searchmain

@markjohngraham , @rchrd2  kindly review this pull request

**Pull request closed without merging**